app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
   belongs_to :post
+
+  after_save :update_like_counter
+
+  def update_like_counter
+    post.increment_counter(:likes_counter, 1)
+  end
```

Comment:
- [ ] As I mentioned above, I think it would be better if your use the ``increment!`` method to update the comments counter as the current one doesn't work on my side. you can update it to something like: ``post. increment!(::likes_counter)``